src/test/resources/login.properties

```diff
@@ -0,0 +1,3 @@
+url=https://mail.yandex.com
+userName=fine.lname
```

Comment:
It will be better to right properties names in a bit another way:
user.name
user.password

Reply:
Done. Is that from a best practice?